php/commands/search-replace.php

```diff
@@ -91,6 +91,13 @@ class Search_Replace_Command extends WP_CLI_Command {
 	 *
 	 *     # Search/replace to a SQL file without transforming the database
 	 *     wp search-replace foo bar --export=database.sql
+	 * 
+	 *     # Search/replace production to development url (multisite compatible)
+	 *     if $(wp --url=http://example.com core is-installed --network); then
```

Comment:
What do you think about having some indication that this is a bash script meant to be run?